test/common/config.spec.ts

```diff
@@ -74,6 +74,7 @@ describe('extractNamespaceName()', () => {
     expect(config.HTTPS_PROXY).toBeUndefined();
     expect(config.HTTP_PROXY).toBeUndefined();
     expect(config.NO_PROXY).toBeUndefined();
+    expect(config.USE_KEEPALIVE).toEqual(true);
```

Comment:
WDYT of adding a similar check to the integration test `kubernetes.spec.ts` where we verify at runtime the env vars, volumes, securityContext, etc.